lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1910,6 +1925,21 @@ bool LLVMToSPIRV::isKnownIntrinsic(Intrinsic::ID Id) {
   }
 }
 
+SPIRVInstruction *LLVMToSPIRV::applyRoundingModeConstrain(Value *V,
```

Comment:
Please add a comment (probably in Doxygen style) describing this function (specify that `V` is `metadata <rounding mode>` argument of `llvm.experimental.constrained.*` intrinsics and this function is designed to map that rounding mode into SPIR-V rounding mode)